src/Skoruba.IdentityServer4.Admin.EntityFramework.Identity/Repositories/IdentityRepository.cs

```diff
@@ -360,6 +360,53 @@ public virtual async Task<IdentityResult> DeleteUserAsync(string userId)
             return await UserManager.DeleteAsync(userIdentity);
         }
 
+        public virtual async Task<int> DeleteClaimForRolesAsync(List<string> roleIds, string roleClaimType)
+        {
+            var rolesQ = from r in DbContext.Set<TRole>()
+                join rc in DbContext.Set<TRoleClaim>() on r.Id equals rc.RoleId
+                where rc.ClaimType.Equals(roleClaimType) && roleIds.Contains(r.Id.ToString())
```

Comment:
roleIds.Contains(r.Id.ToString()) in EFCore3.1 is not translated.

try: roleIds.Contains(((string)(object)r.Id))